lib/internal/repl.js

```diff
@@ -69,6 +66,11 @@ function createRepl(env, opts, cb) {
 }
 
 function setupHistory(repl, historyPath, ready) {
+  // Lazy load
+  const fs = require('fs');
+  const path = require('path');
```

Comment:
`path` is definitely always loaded eager and I do not see how we can prevent that as it is necessary during bootstrapping. So this should probably not be lazy loaded.